### PR TITLE
Lists modal toast rating

### PR DESCRIPTION
**Pull request by prankurpandeyy**

### CODE REVIEW

**Review by Naman-Saxena1 — COMMENTED** [review]
In Modal, close button is not horizontally aligned with text.
Rating is a static image, maybe later you can add some functionality to it.
And you need to work a little on UI part.

File: components/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    /* z-index: 1; Sit on top */
+    padding-top: 100px; /* Location of the box */
+    /* left: 0; */
+    /* top: 0; */
+    width: 100%; 
+    /* Full width */
+    height: 100%; 
+    /* Full height */
+    /* overflow: auto; Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
```
Comment (nikhiljugale007) [inline components/Modal/modal.css:14]: remove commented code that is not necessary.

**Comment by nikhiljugale007** [comment]
In modal you should also give modal title.  Things are not aligned in modal.
Rating is static.
added comments in code.

File: components/Modal/modal.css (hunk 1 above)
Comment (prankurpandeyy) [reply components/Modal/modal.css:14]: sure bro will do that now.

**Comment by prankurpandeyy** [comment]
@Naman-Saxena1  thanks for the review brother , I will update the code .

**Pull request merged**